src/Jackett.Common/Definitions/mononokebt.yml

```diff
@@ -17,28 +19,46 @@ caps:
     tv-search: [q, season, ep]
 
 settings:
-  - name: username
+  - name: cookie
     type: text
-    label: Username
-  - name: password
-    type: password
-    label: Password
+    label: Cookie
+  - name: info
+    type: info
+    label: How to get the Cookie
+    default: "<ol><li>Login to this tracker with your browser<li>Open the <b>DevTools</b> panel by pressing <b>F12</b><li>Select the <b>Network</b> tab<li>Click on the <b>Doc</b> button (Chrome Browser) or <b>HTML</b> button (FireFox)<li>Refresh the page by pressing <b>F5</b><li>Click on the first row entry<li>Select the <b>Headers</b> tab on the Right panel<li>Find <b>'cookie:'</b> in the <b>Request Headers</b> section<li><b>Select</b> and <b>Copy</b> the whole cookie string <i>(everything after 'cookie: ')</i> and <b>Paste</b> here.</ol>"
   - name: vostfr
     type: checkbox
     label: Add VOSTFR to titles
     default: false
+  - name: IMPORTANT_FR
+    type: info
+    label: (FR) Informations Importantes
+    default: Avant de pouvoir télécharger un torrent depuis Jackett, allez dans la <a href="https://mononoke-bt.org/faq.php" target="_blank">FAQ</a> et cliquer tous les 'Lu et Approuvé' de la page.
+  - name: IMPORTANT_EN
+    type: info
+    label: (EN) Important infos
+    default: Before you can even begin to download anything you need to go to the <a href="https://mononoke-bt.org/faq.php" target="_blank">FAQ</a> and click every 'Lu et Approuvé' present on the page.
 
 login:
   path: takelogin.php
-  method: post
+  method: cookie
   inputs:
-    username: "{{ .Config.username }}"
-    password: "{{ .Config.password }}"
+    cookie: "{{ .Config.cookie }}"
+  test:
+    path: index.php
+    selector: a[href="logout.php"]
   error:
     - selector: td.embedded:has(h2:contains("l'authentification"))
   test:
     path: browse2.php
```

Comment:
you've now got two sets of test in this login block. only one will be implemented, I suspect its the last one, instead of the first that is probably the one intended.
the path statement and error statement are redundant when using the method: cookie.
please correct.